isotovideo

```diff
@@ -116,8 +116,16 @@ use commands;
 
 $bmwqemu::vars{BACKEND} ||= "qemu";
 
-# Load the main.pm from the casedir checked by the sanity checks above
-require $bmwqemu::vars{CASEDIR}."/main.pm";
+# Try to load the main.pm from either casedir checked by the sanity checks
+# above or fall back to load from a "distri" subdirectory to allow test
+# distribution collections with multiple distributions or flavors in one
+# repository.
+if (-f "$bmwqemu::vars{CASEDIR}/main.pm") {
+    require $bmwqemu::vars{CASEDIR} . "/main.pm";
+}
+else {
+    require $bmwqemu::vars{CASEDIR} . "/products/$bmwqemu::vars{DISTRI}/main.pm";
+}
```

Comment:
As discussed with coolo: Should be
```
IF $TEST_VARIANT
ELSEIF $DISTRI-$VERSION
ELSEIF $DISTRI
ELSE $CASEDIR
```
and use path as resolved internally to load needles from same dir